apps/remix-ide/src/app/editor/contextView.js

```diff
@@ -15,7 +15,7 @@ const css = require('./styles/contextView-styles')
 class ContextView {
   constructor (opts, localRegistry) {
     this._components = {}
-    this._components.registry = localRegistry || globalRegistry
+    this._components.registry = localRegistry || Registry.getInstance()
```

Comment:
I guess you can now remove `localRegistry`  everywhere (this is basically the same as the global one)